src/view/com/composer/labels/LabelsBtn.tsx

```diff
@@ -23,7 +25,7 @@ export const LabelsBtn = observer(function LabelsBtn({
     <Button
       type="default-light"
       testID="labelsBtn"
-      style={styles.button}
+      style={[styles.button, !hasMedia && styles.dimmed]}
```

Comment:
We can use the `disabled` prop on the `Pressable` component too

Reply:
It's actually not supposed to disable, the dim is just a hint. Tapping on it then gives more info.